package.json

```diff
@@ -21,7 +21,7 @@
   "types": "node/lib/index.d.ts",
   "files": [
     "npm-scripts.js",
-    "node/lib",
+    "node/lib/",
```

Comment:
I don't think `/` is necessary at the end

Reply:
It's not. I just love to differentiate files and folders :)

Reply:
I see. I prefer the opposite because it plays nicely with various kinds of concatenation:
```javascript
const dir = 'node/lib';
const file = 'test.js';
const fullPath = `${dir}/${file}`;
```

When you have extra `/` at the end you'll get `//`, which sometimes works and sometimes causes problems.

Reply:
Done in https://github.com/versatica/mediasoup/pull/954/commits/a962fdea7bdb13101638aa83aea7e036b2fda454